modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -408,6 +106,32 @@ For the nodes within a cluster, additional information from ``skuba`` and the {k
 
 For more information about the data available for nodes, see https://documentation.suse.com/suse-caasp/4/html/caasp-admin/_cluster_updates.html.
 
+.Procedure: Prepare the Configuration Credentials to the Management Node
+
+You will need to prepare the configuration from your cluster to the management node:
+
+. Copy the ``skuba`` configuration directory from your cluster to the {productname} Server, and take a note of the file location.
+// Default file location? --LKB 2020-06-04
+This is the directory that the ``skuba`` service creates after the cluster has been bootstrapped.
+
+. Alternatively:
+  * Copy the passwordless private SSH key used to access the cluster nodes to the {productname} Server, and take a note of the file location.
+You need the current keys, and keys for any clients that you want to use in the future.
+  * You can use an ``ssh-agent`` socket, and provide the path to the socket when setting up the cluster. There are two ways of using the ``ssh-agent`` with {caasp}:
+
+    ** By using ``ssh-agent`` locally:
+    *** Start the ssh-agent locally: ``eval $(ssh-agent)``
+    *** Add the SSH key: ``ssh-add <key>``
+    *** The socket used to access the agent is available in the ``$SSH_AGENT`` environment variable.
+
+    ** Forward the `ssh-agent` to the management node from another machine:
+
+      *** From your source machine: ``ssh -A <management node>``. The socket path is also available in the ``$SSH_AGENT`` environment variable.
```

Comment:
This isn't really a procedure, it's three alternative options, so let's phrase it like that:

```suggestion
You need to prepare the configuration between the cluster and the management node:
There are three ways you can achieve this, choose the method that best suits your environment:

* Copy the ``skuba`` configuration directory from your cluster to the {productname} Server.
This is the directory that the ``skuba`` service creates after the cluster has been bootstrapped.
Take a note of the new file location for adding the cluster in the {productname} {webui}.
* Copy the passwordless private SSH key used to access the cluster nodes to the {productname} Server.
You need the current keys, and keys for any clients that you want to use in the future.
Take a note of the new file location for adding the cluster in the {productname} {webui}.
* Use an ``ssh-agent`` socket, and provide the path to the socket when setting up the cluster. 

If you choose to use an ``ssh-agent`` socket, there are two methods:
To use ``ssh-agent`` locally, start the ``ssh-agent`` on the local client with [command]``eval $(ssh-agent)``, and add the SSH key with [command]``ssh-add <key>``.
The socket used to access the agent is in the ``$SSH_AGENT`` environment variable.

Alternatively, forward the ``ssh-agent`` to the management node from another machine.
From your source machine use [command]``ssh -A <management node>``.
The socket used to access the agent is in the ``$SSH_AGENT`` environment variable.
```

Reply:
Fixed